gapic-common/lib/gapic/rest/grpc_transcoder.rb

```diff
@@ -108,11 +108,10 @@ def transcode request
           uri_values = bind_uri_values! http_binding, request_hash
           next if uri_values.any? { |_, value| value.nil? }
 
-          if http_binding.body && http_binding.body != "*"
+          if http_binding.body && http_binding.body != "*" && !(request.respond_to? http_binding.body.to_sym)
```

Comment:
Optional style nit: You could just say `next if http_binding...`

Reply:
Rubocop fixed that one to what it is right now